Millestone 1: Skeleton, Header and Headline

GuilhermeGiachellin opened the pull request:
-The first addition to the project.
-Completed the skeleton so the code can be expanded easily and more organized.
-Header and Headline have a complete structure and style.

GuilhermeGiachellin committed:
Add Linter

-Added the Html and CSS Linter

GuilhermeGiachellin committed:
Assets folder addition

-All the assets needed for project are added.

GuilhermeGiachellin committed:
Create index file

-Index file with the body of the project.
-All the written content is present.
-Links with GoogleFont information were added.

GuilhermeGiachellin committed:
Create style file

-All the style is present.
-All the formatation is present.

GuilhermeGiachellin committed:
Update style file

-Update to pass the liter

GuilhermeGiachellin committed:
Update style

-Linters padronization

GuilhermeGiachellin committed:
Update style.css

GuilhermeGiachellin committed:
Update style.css

GuilhermeGiachellin committed:
Update style.css

GuilhermeGiachellin committed:
Linters update

mricanho commented on index.html:
```diff
@@ -0,0 +1,50 @@
+</head>
+
+<body>
+    <div class="container">
```
- [x] This div tag is unnecessary, you can just use the header tag instead.

descholar-ceo replied:
:+1:  Issue addressed successfully :heavy_check_mark:

mricanho commented on index.html:
```diff
@@ -0,0 +1,50 @@
+            <div class="toolbar">
+                <div class="logo">
+                    <h2>My Logo</h2>
+                </div>
```
- [x] The logo must have a link (a tag)

descholar-ceo replied:
:+1: Issue addressed successfully :heavy_check_mark:

mricanho commented on index.html:
```diff
@@ -0,0 +1,50 @@
+                <div class="logo">
+                    <h2>My Logo</h2>
+                </div>
+                <div class="menu"><img src="assets/Union.png" alt="Menu"></div>
+            </div>
```
- [x] The hamburger button must have a `nav` tag

descholar-ceo replied:
:+1: Issue addressed successfully :heavy_check_mark:  

- [ ] [Optional] You can just use [fontawesome](https://fontawesome.com/v4.7.0/icons/) icons, instead of a bare image, when you use icons, you page size is optimized hence it increases the performance

mricanho commented on index.html:
```diff
@@ -0,0 +1,50 @@
+                <div class="menu"><img src="assets/Union.png" alt="Menu"></div>
+            </div>
+        </header>
+        <div class="content">
+            <h1>I’m Guilherme Glad to see you!</h1>
+            <div>
+                <p>
+                    I’m a software developer! I can help you build a product , feature or website Look through some of
+                    my work and experience! If you like what you see and have a project you need coded, don’t hestiate
+                    to contact me.
+                </p>
+            </div>
+            <div>
+                <h2><a href="#">Let's Connect</a></h2>
+            </div>
+            <div class="social">
+                <a href="#"><img src="assets/Twitter.png" alt="Twitter"></a>
+                <a href="#"><img src="assets/Linkedin.png" alt="Linkedin"></a>
+                <a href="#"><img src="assets/M.png" alt="M"></a>
+                <a href="#"><img src="assets/Git.png" alt="GitHub"></a>
+                <a href="#"><img src="assets/Hand.png" alt="Hand"></a>
+            </div>
+        </div>
```
- [x] The container of the section is a `section` tag, please change it

descholar-ceo replied:
:+1: Issue addressed successfully :heavy_check_mark:

mricanho commented on index.html:
```diff
@@ -0,0 +1,50 @@
+            <div>
+                <h2><a href="#">Let's Connect</a></h2>
+            </div>
+            <div class="social">
+                <a href="#"><img src="assets/Twitter.png" alt="Twitter"></a>
+                <a href="#"><img src="assets/Linkedin.png" alt="Linkedin"></a>
+                <a href="#"><img src="assets/M.png" alt="M"></a>
+                <a href="#"><img src="assets/Git.png" alt="GitHub"></a>
+                <a href="#"><img src="assets/Hand.png" alt="Hand"></a>
+            </div>
```
- [x] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag), please change it

descholar-ceo replied:
:+1: Issue addressed successfully :heavy_check_mark:

mricanho commented on index.html:
```diff
@@ -0,0 +1,50 @@
+                </p>
+            </div>
+            <div>
+                <h2><a href="#">Let's Connect</a></h2>
```
- [x] The button is a link `a` or `button` tag, and follow the button styling: Active, Hover and Pressed (see design)

descholar-ceo replied:
:+1: Issue addressed successfully :heavy_check_mark:

mricanho reviewed (changes requested):
Hi @GuilhermeGiachellin, 

Good job so far! 🔥🔥 You followed the gitflow and your commit messages are consistent, nice work on your linters configuration.

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

- **For future projects it's important to have the README file in the branch you want to merge (feature branch), not the branch you want to merge into (main branch), in another project this could lead to the invalidation of the project. Take note.**

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

![shine team](https://user-images.githubusercontent.com/60631456/120593837-acec9b00-c405-11eb-8f50-91a4d0737cdf.gif)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

GuilhermeGiachellin committed:
Update index.html

-Changed <div> that has the content inside for a section.
-Added <nav> tag to the hamburguer menu.
-Created a list for the social midia.

GuilhermeGiachellin committed:
Update style.css

-Added a hover and active for the <a> tag.
-Stylized the list.

GuilhermeGiachellin committed:
Update style.css

GuilhermeGiachellin committed:
Update style.css

descholar-ceo reviewed (approved):
Hi @GuilhermeGiachellin :handshake: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :tada: 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
